code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
https://github.com/code-dot-org/code-dot-org/tree/pr_7181 I made a branch based off of this one using browserify-global-shim, and afaict things are working as expected for me. What isn't working for you?